.ci/.jenkins-loggers.yml

```diff
@@ -0,0 +1,4 @@
+---
+loggers:
+  - REPO: "ecs-logging-java"
+    SPEC_PATH: "ecs-logging-core/src/test/resources/spec"
```

Comment:
Did you want data for the other logging repos now? Or separately?  Currently they are:

```
./ecs-logging-go-logrus/internal/spec/v1.json
./ecs-logging-go-zap/internal/spec/v1.json
./ecs-logging-java/ecs-logging-core/src/test/resources/spec/spec.json
./ecs-logging-js/utils/ecs-logging/spec.json
./ecs-logging-python/tests/resources/spec.json
./ecs-dotnet/tests/Elastic.CommonSchema.Tests/Specs/spec.json
./ecs-logging-ruby/spec/fixtures/spec/spec.json
```

Note that ecs-logging-js is going to change names to ecs-logging-nodejs soonish as well.